quarkus-pact-consumer/pom.xml

```diff
@@ -23,6 +24,11 @@
           <artifactId>quarkus-maven-plugin</artifactId>
           <version>${quarkus.version}</version>
         </plugin>
+        <plugin>
+          <groupId>io.quarkus</groupId>
+          <artifactId>quarkus-config-doc-maven-plugin</artifactId>
+          <version>${quarkus.version}</version>
+        </plugin>
```

Comment:
Sorry, missed the ping, just saw it now. I don't think this will fly, except if I misunderstand your project structure? The plugin is only used in the `docs` module so you only need to define the version there... on in the global parent POM.

Reply:
Oh, rats. I should have re-pinged but I decided to try and save you time. :(

Reply:
Hmm, I've re-read https://github.com/quarkusio/quarkus/wiki/Migration-Guide-3.14#for-extension-developers, and not sure I'm understanding what I need to do. 

This is the line I was trying to imitate:

```
diff --git a/pom.xml b/pom.xml
index c87d42b..10cbacd 100644
--- a/pom.xml
+++ b/pom.xml
@@ -98,6 +98,11 @@
                     <artifactId>quarkus-maven-plugin</artifactId>
                     <version>${quarkus.version}</version>
                 </plugin>
+                <plugin>
+                    <groupId>io.quarkus</groupId>
+                    <artifactId>quarkus-config-doc-maven-plugin</artifactId>
+                    <version>${quarkus.version}</version>
+                </plugin>
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-compiler-plugin</artifactId>
                     
 ```
 
 That change is made in the root pom, not the docs pom. My project has two sub-projects, each holding an extension, so I added `quarkus-config-doc-maven-plugin` into the root pom for the sub-projects. The root pom for the repo doesn't do things like define `maven-compiler-plugin`, so it didn't seem to fit. 
 
 @gsmet, do you have some recommendations for how I would I know if it's working?

Reply:
The idea is to add it there:

https://github.com/quarkiverse/quarkus-pact/blob/57f344757778b62f9b4b1461b419d349c6ec5615/pom.xml#L44-L52

Together with the `quarkus-maven-plugin` version. It's just the `pluginManagement` so it will just make sure the version is set when the plugin is used in your submodules.
You can also add the version only when the plugin is declared but I think it's better to be consistent with where we set the `quarkus-maven-plugin` version.

Your extension is not exactly a typical Quarkus extension created with the template, that's why things are a bit different for you.

Reply:
Thank you! I wish poms were smaller and more readable :(